### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by 0904-mansi**
in this commit, I have added Beginner-friendly issues and how to solve them.

### CODE REVIEW

File: content/en/docs/CommunityLeaders/EventHandbooks/Docker101/gettingstarted/_index.md [hunk 1]
```diff
@@ -82,6 +82,71 @@ For more examples and ideas, visit:
  https://docs.docker.com/get-started/
 ```
 
+###  If You Are Facing This Error
```
Comment (gauravgahlot) [inline content/en/docs/CommunityLeaders/EventHandbooks/Docker101/gettingstarted/_index.md:85]:
```suggestion
## Troubleshooting
```

File: content/en/docs/CommunityLeaders/EventHandbooks/Docker101/gettingstarted/_index.md [hunk 2]
```diff
@@ -82,6 +82,71 @@ For more examples and ideas, visit:
...
+  ```
+$ docker run hello-world
+```
+```
+docker: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/create": dial unix /var/run/docker.sock: connect: permission denied.
+See 'docker run --help'.
+```
+
+
+## Add "sudo" Before Run Any Docker Command
```
Comment (gauravgahlot) [inline content/en/docs/CommunityLeaders/EventHandbooks/Docker101/gettingstarted/_index.md:95]: While both the solutions you have provided will work, but these are not recommended. You may want to check out this document on the Docker website: [https://docs.docker.com/engine/install/linux-postinstall/](https://docs.docker.com/engine/install/linux-postinstall/).

**Review by gauravgahlot — CHANGES_REQUESTED** [review]
Welcome to the community @0904-mansi. :rocket: 

I have provided some feedback on the changes. Please feel free to put out any questions you may have. Thanks.

**Comment by ajeetraina** [comment]
@0904-mansi Most of these tutorials are PWD based and we don't need sudo fo such environments.

**Comment by 0904-mansi** [comment]
> @0904-mansi Most of these tutorials are PWD based and we don't need sudo fo such environments.

ok sir ☹

**Pull request closed without merging**